plugin-locationcomponent/api/metalava.txt

```diff
@@ -0,0 +1,76 @@
+// Signature format: 3.0
```

Comment:
can we somehow divide this file into packages?
for more readability and not so often updates

Reply:
I didn't find the option to divide the metalava generated file, the output is per module.